py_vsys/api.py

```diff
@@ -443,6 +443,18 @@ async def get_tok_info(self, tok_id: str) -> Dict[str, Union[int, str]]:
         """
         return await self._get(f"/tokenInfo/{tok_id}")
 
+    async def get_last_index(self, ctrt_id: str) -> Dict[str, Union[int, str]]:
+        """
+        get_last_index gets the last index of the NFT contract.
```

Comment:
```suggestion
        get_last_index gets the last index of the token in a token-defining contract(e.g. NFT contract, token contract).
```
Just quickly verified, this api works for token-defining contracts(e.g. NFT, token contract).
For token-operating contract (e.g. lock contract), it returns soemthing like below

```
{
  "error": 199,
  "message": "No token generated in this contract"
}
```